src/Hydrator/Config/Workflow/WorkflowSchemaVersionEnum.php

```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
```

Comment:
At the moment this makes no sense for me - the version number is always indicated in the client itself.

If there is a new API version, there must also be a new client, with new methods and a new version number.